app/main.py

```diff
@@ -2,24 +2,68 @@
 
 
 class IntegerRange:
-    pass
+
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __get__(self, instance: object, owner: object) -> object:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if self.min_amount <= value <= self.max_amount:
+            self.protected_name = value
+            return
+        raise ValueError
```

Comment:
I think you can change condition to opposite and raise `ValueError` inside `if`. And outside `if` you can set the value for `protected_name`. This way we can avoid redundant return

Reply:
Great comment, this way code looks much nicer, thanks!